ideainspect.groovy

```diff
@@ -242,6 +261,59 @@ private File findIdeaExecutable(OptionAccessor cliOpts) {
   ideaExecutable
 }
 
+private modifyIdeaProps(OptionAccessor cliOpts){
+  println "#"
```

Comment:
I thought that was what this did. It first copies the file to idea.properties.copy in modifyIdeaProps and then edits idea.properties.copy. Then in cleanUpIdeaProps it deletes that and restores the copy. Or is not the right way of doing it?